openupgrade_scripts/apriori.py

```diff
@@ -45,6 +45,7 @@
     "sale_project_account": "sale_project",
     "website_sale_delivery_giftcard": "website_sale_loyalty_delivery",
     "website_sale_gift_card": "website_sale_loyalty",
+    "pad": "web_editor",
```

Comment:
Please, put above pos_coupon to maintain alphabetical order.

Reply:
Done sir, i'll take note for this for other relevant PR